### DISCUSSION TITLE
Subreddit Ownership

### DISCUSSION THREAD

**Discussion by KaraMartin** [tenacityteam/tenacity-legacy#463] [General]
Hey there, this is unrelated to the software proper, but I made the subreddit r/tenacity a long time ago since it's a term related to League of Legends, but never ended up using it. I've had a couple people message me on reddit about it, but wanted to come to the source to make sure it ended up in the right hands. So if any of the devs have a reddit account I'd be happy to transfer that to ya!

**Comment by emabrey** [comment]
[/u/tenacityaudio](https://www.reddit.com/user/TenacityAudio/overview) is the official account.

**Reply by Semisol** [reply]
I don't think it has sufficient karma.

**Reply by NotAProton** [reply]
post memes I guess

to get karma of course
